learn/user-guide/getting-started/Installation-options.md

```diff
@@ -88,8 +88,7 @@ You need to download and install the below to build the Ballerina modules.
     
     >**Info:** You can also use [Oracle JDK](https://www.oracle.com/java/technologies/javase-downloads.html). Set the JAVA_HOME environment variable to the pathname of the directory into which you installed JDK.
 
-2. [Node.js (version 8.9.x)](https://nodejs.org/en/download/)
-3. [npm (version 5.6.0 or later)](https://www.npmjs.com/get-npm)
+2. [npm (version 5.6.0 or later)](https://www.npmjs.com/get-npm)
```

Comment:
@praneesha Lets remove npm as well. This is also not required for ballerina-lang

Reply:
Done.